platform-sdk/platform-apps/tests/StatsSigningTestingTool/src/main/java/com/swirlds/demo/stats/signing/SttTransactionPool.java

```diff
@@ -63,7 +63,7 @@ final class SttTransactionPool {
     private final boolean signed;
 
     /**
-     * the standard psuedo-random number generator
+     * the standard pseudo-random number generator
```

Comment:
I'm sensing a bit of cut-and-paste happened with these ... not that there's anything wrong with that ...